### DISCUSSION TITLE
Local executor: `Cannot find module '@project/my-plugin/package.json'`

### DISCUSSION THREAD

**Discussion by SimonCockx** [prisma/prisma#16639] [Q&A] [answered]
I created a local executor, but when I try to run it in one of my projects, it fails with
```
Unable to resolve @project/my-plugin:my-executor.

Cannot find module '@project/my-plugin/package.json'
```

I got this with the following steps. 

1. Generate the plugin.
```
nx g @nrwl/nx-plugin:plugin my-plugin
```
I removed the default generator and executor.

2. Generate the executor.
```
nx generate @nrwl/nx-plugin:executor my-executor --project=my-plugin
```

3. Add it as a target in another project.
```
{
  ...
  "targets": {
    ...
    "my-executor": {
      "executor": "@project/my-plugin:my-executor" 
    }
  }
}
```

Any idea what went wrong?

**Comment by SimonCockx** [comment, marked as answer]
Sorry, although my actual use case is prisma related, I should ask this in the `nx` repo. Moving to https://github.com/nrwl/nx/discussions/13649.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
